api/conf/conf.yaml

```
@@ -66,8 +66,7 @@ conf:
   #   access_control_allow_headers: "Authorization"
   #   access_control-allow_methods: "*"
   #   x_frame_options: "deny"
-  #   content_security_policy: ""default-src 'self'; script-src 'self' 'unsafe-eval'; style-src 'self' 'unsafe-inline'""
-
+  #   content_security_policy: "default-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; frame-src xx.xx.xx.xx:3000"  # You can set frame-src to provide content for your grafana panel.
```

This line of text is too long, and you can't guarantee that everyone's editor is set to auto wrap line.

```suggestion
  #   You can set frame-src to provide content for your grafana panel.
  #   content_security_policy: "default-src 'self'; script-src 'self' 'unsafe-eval' 'unsafe-inline'; style-src 'self' 'unsafe-inline'; frame-src xx.xx.xx.xx:3000" 
```

---

Not recommended, based on the style of the whole document, I think it would be misleading back to the user if it did.

If you insist, I suggest inviting @bzp2010  to make the final judgment.

---

I think it's acceptable for now, and when we merge it to the `next` branch it will be modified according to the new format. 🤔